packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -147,240 +104,459 @@ type TileProps = {
+TileContentArea.displayName = 'Tile.ContentArea';
+
+// ----------------------------------------------------------------------------
+// Tile.Description component
+
+const TileDescription = polymorphic('iui-tile-description');
+TileDescription.displayName = 'Tile.Description';
+
+// ----------------------------------------------------------------------------
+// Tile.Metadata component
+
+const TileMetadata = polymorphic('iui-tile-metadata');
+TileMetadata.displayName = 'Tile.Metadata';
+
+// ----------------------------------------------------------------------------
+// Tile.MoreOptions component
+type TileMoreOptionsOwnProps = {
+  onButtonClick?: React.MouseEventHandler<HTMLElement>;
+  children?: React.ReactElement[];
 };
 
+const TileMoreOptions = React.forwardRef((props, forwardedRef) => {
+  const { className, children = [], onButtonClick, ...rest } = props;
+  const [isMenuVisible, setIsMenuVisible] = React.useState(false);
+
+  return (
+    <DropdownMenu
+      onShow={React.useCallback(() => setIsMenuVisible(true), [])}
+      onHide={React.useCallback(() => setIsMenuVisible(false), [])}
+      menuItems={(close) =>
+        children?.map((option: React.ReactElement) =>
+          React.cloneElement(option, {
+            onClick: (value: unknown) => {
+              close();
+              option.props.onClick?.(value);
+            },
+          }),
+        )
+      }
+    >
+      <Box
+        className={cx(
+          'iui-tile-more-options',
+          {
+            'iui-visible': isMenuVisible,
+          },
+          className,
+        )}
+        ref={forwardedRef}
+        {...rest}
+      >
+        <IconButton
+          styleType='borderless'
+          size='small'
+          aria-label='More options'
+          onClick={onButtonClick}
+        >
+          <SvgMore />
+        </IconButton>
+      </Box>
+    </DropdownMenu>
+  );
+}) as PolymorphicForwardRefComponent<'div', TileMoreOptionsOwnProps>;
+TileMoreOptions.displayName = 'Tile.MoreOptions';
+
+// ----------------------------------------------------------------------------
+// Tile.Buttons component
+
+const TileButtons = polymorphic('iui-tile-buttons');
+TileButtons.displayName = 'Tile.Buttons';
+
+/**
+ * Tile with customizable Thumbnail, Name, Content and Buttons subcomponents
+ * @example
+ * <Tile>
+ *    <Tile.ThumbnailArea>
+ *      <Tile.ThumbnailPicture/>
+ *      <Tile.Badge/>
+ *      <Tile.TypeIndicator/>
+ *      <Tile.QuickAction/>
+ *    </Tile.ThumbnailArea>
+ *    <Tile.Name>
+ *      <Tile.NameIcon/>
+ *      <Tile.NameLabel/>
+ *    <Tile.Name/>
+ *    <Tile.ContentArea>
+ *      <Tile.Description />
+ *      <Tile.Metadata/>
+ *      <Tile.MoreOptions/>
+ *    </Tile.ContentArea>
+ *    <Tile.Buttons/>
+ * </Tile>
+ */
+export const Tile = Object.assign(TileComponent, {
+  /**
+   *ThumbnailArea subcomponent that contains `ThumbnailPicture`, `QuickAction`, `TypeIndicator` or `BadgeContainer`
```

Comment:
I'd add a space before `ThumbnailArea` on this line

Reply:
done 😄 , and thanks!